source/features/latest-tag-button.tsx

```diff
@@ -111,21 +107,29 @@ async function init(): Promise<false | void> {
 		link.append(' ', <span className="css-truncate-target">{latestTag}</span>);
 	}
 
-	if (currentBranch === latestTag || isUpToDate) {
+	if (currentBranch === latestTag || aheadBy === 0) {
 		link.setAttribute('aria-label', 'You’re on the latest release');
 		link.classList.add('disabled', 'tooltipped', 'tooltipped-ne');
 		return;
 	}
 
 	const defaultBranch = await getDefaultBranch();
 	if (currentBranch === defaultBranch) {
-		const aheadBy = await getAheadByCount(latestTag);
-		if (!aheadBy) {
-			return;
+		link.append(<sup> +{aheadBy ?? ''}</sup>);
```

Comment:
```suggestion
		link.append(<sup> +{aheadBy}</sup>);
```

Reply:
Cant do that

![image](https://user-images.githubusercontent.com/16872793/84184649-c1e55700-aa5b-11ea-97f5-3298ad1b4cd7.png)

Reply:
That's weird, it's supposed to work: https://github.com/vadimdemedes/dom-chef/blob/1365a05fa7ce343082cb34caf9dd2717ac32b728/index.ts#L68-L69

Reply:
And indeed it works in the tests: 
https://github.com/vadimdemedes/dom-chef/commit/a87f8ed6159e2e93522e75a611e2ca213bb84674
https://travis-ci.org/github/vadimdemedes/dom-chef/builds/696577490

Reply:
Would tagged template made a diff?

Reply:
Very sorry I was 100% wrong.
I did not notice you took off the tagged template